src/components/board/client.test.ts

```diff
@@ -13,6 +13,47 @@ import { BoardServiceClient } from '../../gen/component/board/v1/board_pb_servic
 vi.mock('../../gen/component/board/v1/board_pb_service');
 
 let board: BoardClient;
+const testAnalogMin = 0;
+const testAnalogMax = 5;
+const testStepSize = 0.1;
+const testValue = 2.2;
+
+const testAnalogValue: AnalogValue = {
+  value: testValue,
+  minRange: testAnalogMin,
+  maxRange: testAnalogMax,
+  stepSize: testStepSize,
+};
+
+beforeEach(() => {
+  RobotClient.prototype.createServiceClient = vi
+    .fn()
+    .mockImplementation(() => new BoardServiceClient('mysensor'));
+
+  BoardServiceClient.prototype.readAnalogReader = vi
+    .fn()
+    .mockImplementation((_req, _md, cb) => {
+      cb(null, {
+        getValue: () => testValue,
+        getMinRange: () => testAnalogMin,
+        getMaxRange: () => testAnalogMax,
+        getStepSize: () => testStepSize,
```

Comment:
I suspect you no longer need to define these since the client function isn't calling them anymore